rest_server/__main__.py

```diff
@@ -0,0 +1,28 @@
+"""Start server as application."""
+
+import argparse
+import asyncio
+
+from . import server
+from .config import ENV, config_logging
+
+
+async def main() -> None:
+    """Start server."""
+    rs = await server.make()
+    rs.startup(address=ENV.REST_HOST, port=ENV.REST_PORT)  # type: ignore[no-untyped-call]
+    try:
+        await asyncio.Event().wait()
+    finally:
+        await rs.stop()  # type: ignore[no-untyped-call]
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-l", "--log", default="DEBUG", help="the output logging level")
+    _args = parser.parse_args()
```

Comment:
Is the only way to configure logging via the cmd line arg?  This seems odd since we take everything else via env variable.

Reply:
I made it an env variable